src/cli/prompts/retrofit.ts

```
@@ -0,0 +1,377 @@
+    case 'custom':
+      return await prdInput();
+    case 'enhance':
+      const { enhancedPrd } = await inquirer.prompt([
+        {
+          type: 'editor',
+          name: 'enhancedPrd',
+          message: 'Enhance the generated PRD:',
+          default: suggestedPrd,
+        },
+      ]);
+      return { content: enhancedPrd };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)